epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyViewHolder.java

```diff
@@ -43,8 +48,9 @@ public void bind(@SuppressWarnings("rawtypes") EpoxyModel model,
     this.payloads = payloads;
 
     if (epoxyHolder == null && model instanceof EpoxyModelWithHolder) {
-      epoxyHolder = ((EpoxyModelWithHolder) model).createNewHolder();
+      epoxyHolder = ((EpoxyModelWithHolder) model).createNewHolder(parent);
       epoxyHolder.bindView(itemView);
+      parent = null;
```

Comment:
@eboudrant I think this needs to go outside of the braces so that it is nulled out for models that don't have holders as well?

Reply:
Fixed, good catch!

Reply:
Thanks for the quick fix :) I'll merge this, then if you want to get the carousel fix in I can push a release afterwards

Reply:
I am not sure when I will have the carousel fix so having a release seen would be nice (visibility on group and this fix) :)